src/python_testing/support_modules/binfo_attributes_verification.py

```diff
@@ -0,0 +1,285 @@
+            TestStep(9, "TH reads HardwareVersionString from the DUT.",
+                     "Verify it is of type string, and value length in the range of 1 to 64 bytes"),
+            TestStep(10, "TH reads SoftwareVersion from the DUT", "Verify that the value is the range of 0 to 4294967294"),
+            TestStep(11, "TH reads SoftwareVersionString from the DUT.",
+                     "Verify it is of type string, has a length of 1 to 64 bytes"),
+            TestStep(12, "TH reads ManufacturingDate from the DUT",
+                     "Verify it is of type string, length in the range of 8 to 16 bytes, and the first 8 characters specify date according to YYYYMMDD."),
+            TestStep(13, "TH reads PartNumber from the DUT", "Verify it is of type string, and has a max length of 32 bytes"),
+            TestStep(14, "TH reads ProductURL from the DUT.",
+                     "Verify it is of type string, less than or equal to 256 ASCII characters, and follows the syntax rules specified in RFC 3986"),
+            TestStep(15, "TH reads ProductLabel from the DUT.",
+                     "Verify it is of type string, less than or equal to 64 bytes, and does not include the name of the vendor as defined within the VendorName attribute"),
+            TestStep(16, "TH reads SerialNumber from the DUT.", "Verify it is of type string, and has a max length of 32 bytes"),
+            TestStep(17, "TH reads LocalConfigDisabled from the DUT.", "Verify it is of type boolean, and is set to false"),
+            TestStep(18, "TH reads Reachable from the DUT.", "Verify it is of type boolean, and is set to true"),
+            TestStep(19, "TH reads UniqueID from the DUT.",
+                     "Verify it is of type string, and verify that the value is not identical to SerialNumber attribute if SerialNumber attribute is supported"),
+            TestStep(20, "TH reads CapabilityMinima attribute from the DUT",
+                     "Verify that the CaseSessionsPerFabric is in the range of 3 to 65535, and SubscriptionsPerFabric is in the inclusive range of 3 to 65535"),
+            TestStep(21, "TH reads ProductAppearance from the DUT.", "Verify it is of type ProductAppearanceStruct."),
+            TestStep(22, "TH reads SpecificationVersion from the DUT.",
+                     "Value should be set to a valid Major, Minor, and Dot version with the lower 8 bits set to zero."),
+            TestStep(23, "TH reads MaxPathsPerInvoke from the DUT.", "Verify that the value is in the range of 1 to 65535."),
+        ]
+
+    def pics(self, cluster_pics) -> list[str]:
+        return [f"{cluster_pics}.S"]
+
+    async def implementation(self, cluster: Cluster, cluster_name: str):
+        self.endpoint = self.get_endpoint()
+        self.step(0)  # commissioning already done
+
+        # Step 1: DataModelRevision
+        if cluster_name == "BRBINFO":
```

Comment:
you can get rid of this - the attribute guard will handle this. The attribute just won't be present on bridged device basic info